src/components/PhotoOfTheDay/POTD.module.css

```diff
@@ -0,0 +1,96 @@
+
```

Comment:
CSS modules are good to encapsulate styles for specific components but we agreed to define our styles in objects and pass them to sx prop of MUI components
Move these styling to your empty styles file, please